TRRandomizerCore/Editors/RandomizerSettings.cs

```diff
@@ -211,6 +212,7 @@ public void ApplyConfig(Config config)
         RandomizeItems = config.GetBool(nameof(RandomizeItems));
         ItemSeed = config.GetInt(nameof(ItemSeed), defaultSeed);
         ItemMode = (ItemMode)config.GetEnum(nameof(ItemMode), typeof(ItemMode), ItemMode.Default);
+        WeaponDifficulty = (WeaponDifficulty)config.GetEnum(nameof(WeaponDifficulty), typeof(WeaponDifficulty), WeaponDifficulty.Fair);
```

Comment:
Assuming this is the default, should it be Easy so it is the same as the current behaviour?

Reply:
Good point. I'll change it to easy, in case people overlook it.